lib/registrant.rb

```diff
@@ -0,0 +1,53 @@
+class Registrant
+  attr_reader :name, :age, :permit, :license_data
+
+  # Initializes a registrant with a name, age, and permit status
+  # Cannot make this a keyword argument because of the way the workflow
+  # is set up in the iteration instructions.
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = { written: false, license: false, renewed: false }
+  end
+
+  # Changes permit status to true
+  def earn_permit
+    @permit = true
+  end
+
+  # Returns true if registrant has a permit, false otherwise
+  def permit?
+    @permit
+  end
+
+  # Changes written test status to true
+  def administer_written_test
```

Comment:
Nice, I like that you have these setter methods in this class.

Reply:
I don't see tests for all these methods though, make sure we test every helper method and test all possible return values of those methods.